libs/utils/utils.js

```diff
@@ -151,7 +151,8 @@ export const [setConfig, getConfig] = (() => {
       config.autoBlocks = conf.autoBlocks ? [...AUTO_BLOCKS, ...conf.autoBlocks] : AUTO_BLOCKS;
       document.documentElement.setAttribute('lang', config.locale.ietf);
       try {
-        document.documentElement.setAttribute('dir', (new Intl.Locale(config.locale.ietf)).textInfo.direction);
+        const contentDir = getMetadata('content-direction');
+        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || 'ltr');
```

Comment:
@mokimo @narcis-radu Initially the change here was following 
document.documentElement.setAttribute('dir', contentDir || config.locale.dir || (new Intl.Locale(config.locale.ietf)).textInfo.direction);

This was as discussed in the github discussion https://github.com/orgs/adobecom/discussions/746 and did not break the dir property for browsers other that firefox

This was later changed to  contentDir || config.locale.dir || 'ltr' as per the last comment.

Could you please confirm if I should change it to fallback to Intl.Locale(config.locale.ietf)).textInfo.direction so that the teams have time to add the support for Firefox by adding the dir property?

Reply:
@chrischrischris @auniverseaway Could you please share your review on the same?

Reply:
So that we can merge without waiting for consumers:
```suggestion
        document.documentElement.setAttribute('dir', contentDir || config.locale.dir || Intl.Locale(config.locale.ietf))?.textInfo?.direction) || 'ltr');
```

We can remove Intl.Locale once all consumers have updated.

Reply:
@chrischrischris I have made the suggested changes